### PR TITLE
ENH: Log stamping

### PR DESCRIPTION
**Pull request by larsoner**
Adds to the `.log` file that we attempted to stamp some triggers.

### CODE REVIEW

**Comment by larsoner** [comment]
@drammock can you review and merge if you're happy? I think this is a pretty straightforward enhancement

File: expyfun/_experiment_controller.py [hunk 1]
```diff
@@ -1931,11 +1933,10 @@ def _stamp_binary_id(self, id_, delay=0.03, wait_for_last=True):
         if not isinstance(id_, (list, tuple, np.ndarray)):
             raise TypeError('id must be array-like')
         id_ = np.array(id_)
-        if not np.all(np.logical_or(id_ == 1, id_ == 0)):
+        if not np.all((id_ == 1) | (id_ == 0)):
```
Comment (drammock) [inline expyfun/_experiment_controller.py]:
```suggestion
        if not np.all(np.in1d(id_, [0, 1])):
```

Comment (drammock) [reply expyfun/_experiment_controller.py]: I think this is equivalent and maybe slightly clearer.  The current version might be carelessly misread as checking that `id_` is either all 1s or all 0s.

**Comment by larsoner** [comment]
Yes to both of your points. I figure if people want to control the trigger duration and separation separately they can, but assume YAGNI until we hit a use case.

**Pull request merged**